samples/audio-video-player/audio_video_player.html

```diff
@@ -54,7 +54,7 @@
   <canvas width=1280 height=720></canvas>
 </div>
 <script type="module">
-import { WebAudioController } from "./web_audio_controller.js";
+import { WebAudioController } from "../library/web_audio_controller.js";
```

Comment:
Nit: Call the directory `common` or `lib`.

Reply:
Done. Went with lib.